Added issue description and workaround for installer crash problem. (#239)
harvester/docs: docs/install/iso-install.md

   ![first-login.png](/img/v1.1/install/first-time-login.png)

<!-- :::note
In some cases, if you are using an older VGA connector, you may encounter an `panic: invalid dimensions` error with ISO installation. See issue [#2937](https://github.com/harvester/harvester/issues/2937#issuecomment-1278545927) for a workaround.
::: -->

## Known Issue

> ### Installer may crash when using an older graphics card/monitor

Reviewer: ```suggestion
### Installer may crash when using an older graphic card/monitor
```

> In some cases, if you are using an older graphics card/monitor, you may encounter an `panic: invalid dimensions` error with ISO installation.

Reviewer: ```suggestion
In some cases, if you are using an older graphic card/monitor, you may encounter a `panic: invalid dimensions` error during ISO installation.
```

![invalid-dimensions.png](/img/v1.1/install/invalid-dimensions.png)

> This is a known issue we are working on, and will be fixed in the future release. Here is a workaround for this issue:

Reviewer: ```suggestion
This is a known issue we are working on, and will be fixed in future releases. Here is a workaround for this issue:
```

> 1. Boot up with the ISO, and press `E` to edit first menu entry;

Reviewer: ```suggestion
1. Boot up with the ISO, and press `E` to edit the first menu entry:
```

   ![grub-menu.png](/img/v1.1/install/grub-menu.png)

> 2. Append `vga=792` to the line started with `$linux`;

Reviewer: ```suggestion
2. Append `vga=792` to the line started with `$linux`:
```

   ![edit-menu-entry.png](/img/v1.1/install/edit-menu-entry.png)

> 3. Press `Ctrl+X` or `F10` to boot up;

Reviewer: ```suggestion
3. Press `Ctrl+X` or `F10` to boot up.
```
